<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEditDialog</class>
 <widget class="QDialog" name="AddEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>First Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Phone Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>E-mail Address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="firstNameBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lastNameBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="phoneNumBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addCustButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>80</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Add 
Customer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>80</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
